iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/AddInstruction.java

```diff
@@ -19,5 +19,13 @@
  */
 package org.sonar.iac.docker.tree.api;
 
+/**
+ * AddInstruction
+ * This interface define the contract of the <a href="https://docs.docker.com/engine/reference/builder/#add">ADD</> instruction.
+ * It is a strict implementation of the {@link TransferInstruction} interface.
+ * <pre>
+ *   {@link #keyword()} {@link #options()} {@link #srcs()} {@link #dest()}
+ * </pre>
+ */
```

Comment:
I'm wondering if I should in parallel add a literal example, like this :
`ADD --option=value src1 src2 dest`